src/common/pre-packaging-helper.sh

```diff
@@ -34,6 +34,27 @@ setup_maven(){
   export PATH=$M2_HOME/bin:$PATH
 }
 
+
```

Comment:
The util script contains the helper function for maven, java and npm. Hence the name pre-packaging. Are you suggesting to put the npm utility functions to a separate helper file? 
API server changes updated.